juspay/hyperswitch
# feat(payouts): add kafka events

## Type of Change



## Description



### Additional Changes





## Motivation and Context
This helps in viewing payout analytics on HyperSwitch dashboard. Closes a subtask of #3652



## How did you test it?
Env - local
## Steps for testing
- Make sure `events` are set to `kafka`
- Creating a payout transaction should send these events to kafka producer
- In local env, this can be verified from console logs

### Payout create + confirm + auto fulfill
`curl --location 'http://localhost:8080/payouts/create' \
--header 'Content-Type: application/json' \
--header 'api-key: dev_wmZ4BwC6pXMpBZs3kHlEHAbAxQevfCfJvEESnNmpm7eGlRlNH3yqWCjH58Q87Vfm' \
--data-raw '{
    "amount": 1,
    "currency": "EUR",
    "customer_id": "cus_NOWPAviaOF3z7Ynbi5MQ",
    "email": "payout_customer@example.com",
    "name": "John Doe",
    "phone": "999999999",
    "phone_country_code": "+65",
    "description": "Its my first payout request",
    "payout_type": "card",
    "payout_method_data": {
        "card": {
            "card_number": "4111111111111111",
            "expiry_month": "3",
            "expiry_year": "2030",
            "card_holder_name": "John Doe"
        }
    },
    "billing": {
        "address": {
            "line1": "1467",
            "line2": "Harrison Street",
            "line3": "Harrison Street",
            "city": "San Fransico",
            "state": "CA",
            "zip": "94122",
            "country": "US",
            "first_name": "John",
            "last_name": "Doe"
        },
        "phone": {
            "number": "8056594427",
            "country_code": "+91"
        }
    },
    "entity_type": "Individual",
    "recurring": true,
    "metadata": {
        "ref": "123"
    },
    "confirm": true,
    "auto_fulfill": true
}'`
![image](https://github.com/juspay/hyperswitch/assets/46213975/7aca1792-a455-4579-b7ae-46512a7e06d0)


## Checklist
